### PR TITLE
Identification

### PR DESCRIPTION
**Pull request by ThomasMlh**

### CODE REVIEW

**Review by Sarahshr — DISMISSED** [review]
It's ok for me !

File: passport-strategies.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+              const isPasswordOK = bcrypt.compareSync(formPassword, password);
+              if (!isPasswordOK) return done(null, false, "Wrong password!");
+
+              const user = { id, email, firstname, lastname };
+              return done(null, user);
+            }
+          }
+        );
+      } catch (e) {
+        console.log(e);
```
Comment (LoicBrassart) [inline passport-strategies.js]: Nope, no `console.log`

File: routes/auth.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      res.status(201).json(token);
+    } else {
+      res.status(500).json(errSql);
+    }
+  });
+});
+
+router.post("/login", passport.authenticate("local"), (req, res) => {
+  const token = jwt.sign(req.user, jwt_secret);
+  console.log(token);
```
Comment (LoicBrassart) [inline routes/auth.js]: Nope, delete this line

File: routes/misc.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const passport = require("passport");
+const router = express.Router();
+const { db } = require("../conf");
+
+router.get("/", (req, res) => {
+  const msg =
+    "Welcome on Authentication-101! Feel free to read the README.md file";
```
Comment (LoicBrassart) [inline routes/misc.js]: You probably don't need this message, do you ? ;-)

File: routes/misc.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  res.status(200).send(msg);
+});
+
+router.get("/protected", passport.authenticate("jwt"), (req, res) => {
+  const msg =
+    "If you can see this, you should be logged in, " + req.user.firstname;
```
Comment (LoicBrassart) [inline routes/misc.js]: Same here

File: server.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+/* --------------------------------------------------------------------- Routes */
+
+app.use("/auth", require("./routes/auth"));
+app.use("/", require("./routes/misc"));
+
+/* --------------------------------------------------------------------- 404 and server launch */
+app.use((req, res) => {
+  const msg = `Page not found: ${req.url}`;
+  console.warn(msg);
```
Comment (LoicBrassart) [inline server.js]: Nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Many useless code bits, please clean it

**Review by Sarahshr — APPROVED** [review]
It's good for me !

**Review by LoicBrassart — APPROVED** [review]
NmE

**Pull request merged**